test/framework/k8s_controlplane.go

```diff
@@ -79,21 +79,19 @@ func (c *K8sControlPlane) GetKubectlOptions(namespace ...string) *k8s.KubectlOpt
 }
 
 func (c *K8sControlPlane) PortForwardKumaCP() {
-	kumaCpPods := c.GetKumaCPPods()
-	// There could be multiple pods still starting so pick one that's available already
-	for i := range kumaCpPods {
-		if k8s.IsPodAvailable(&kumaCpPods[i]) {
-			c.portFwd.apiServerTunnel = k8s.NewTunnel(c.GetKubectlOptions(Config.KumaNamespace), k8s.ResourceTypePod, kumaCpPods[i].Name, 0, 5681)
-			c.portFwd.apiServerTunnel.ForwardPort(c.t)
-			c.portFwd.ApiServerEndpoint = c.portFwd.apiServerTunnel.Endpoint()
-
-			c.madsFwd.apiServerTunnel = k8s.NewTunnel(c.GetKubectlOptions(Config.KumaNamespace), k8s.ResourceTypePod, kumaCpPods[i].Name, 0, 5676)
-			c.madsFwd.apiServerTunnel.ForwardPort(c.t)
-			c.madsFwd.ApiServerEndpoint = c.madsFwd.apiServerTunnel.Endpoint()
-			return
-		}
+	kumaCpSvc := c.GetKumaCPSvc()
+	if k8s.IsServiceAvailable(&kumaCpSvc) {
+		c.portFwd.apiServerTunnel = k8s.NewTunnel(c.GetKubectlOptions(Config.KumaNamespace), k8s.ResourceTypePod, kumaCpSvc.Name, 0, 5681)
```

Comment:
something like

```suggestion
		c.portFwd.apiServerTunnel = k8s.NewTunnel(c.GetKubectlOptions(Config.KumaNamespace), k8s.ResourceTypeService, kumaCpSvc.Name, 0, 5681)
```

right?

Reply:
right, I was looking at it but I forgot to change it in a hurry